applications/ContactStructuralMechanicsApplication/custom_conditions/paired_condition.cpp

```diff
@@ -119,4 +119,22 @@ void PairedCondition::InitializeNonLinearIteration(const ProcessInfo& rCurrentPr
     KRATOS_CATCH( "" );
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+void PairedCondition::save(Serializer& rSerializer) const
+{
+    KRATOS_SERIALIZE_SAVE_BASE_CLASS( rSerializer, Condition );
+    rSerializer.save("PairedNormal", mPairedNormal);
```

Comment:
it was missing before? this could explain the problem related to the restart in contactsimulations

Reply:
This is only mesh tying, other contact problem require extra changes